lib/shopify-cli/php_deps.rb

```diff
@@ -0,0 +1,99 @@
+require "shopify_cli"
+
+module ShopifyCli
+  ##
+  # ShopifyCli::PHPDeps ensures that all PHP dependencies are installed for projects.
+  #
+  class PHPDeps
+    include SmartProperties
+
+    property! :ctx, accepts: ShopifyCli::Context
+
+    ##
+    # Proxy to instance method ShopifyCli::PHPDeps.new.install.
+    #
+    # #### Parameters
+    # - `ctx`: running context from your command
+    # - `verbose`: whether to run the installation tools in silent mode
```

Comment:
This description is a little weird - if someone doesn't know what "verbose" means, `verbose == true` sounds like "yes, it should be on silent mode".